Source/Krypton Components/Krypton.Toolkit/Palette Builtin/Base/PaletteBase.cs

```diff
@@ -1743,18 +1743,18 @@ public virtual float BaseFontSize
             }
         }
 
-        [AllowNull, DefaultValue(typeof(Font), DEFAULT_FONT_TYPE)]
+        [AllowNull, DefaultValue(typeof(Font), DEFAULT_FONT_STYLE)]
```

Comment:
You have set `AllowNull` -> OK
BUT, then you also have the return type as `null`able AND Check if null in the get and return a non null!

Please remove the `Null`able from the type.

Reply:
Default value is now `null`